homeassistant/components/mqtt/alarm_control_panel.py

```diff
@@ -210,6 +217,7 @@ def message_received(msg):
             ):
                 _LOGGER.warning("Received unexpected payload: %s", msg.payload)
                 return
+            assert isinstance(payload, str)
             self._state = payload
```

Comment:
Can we drop the assert here?
```suggestion
            self._state = str(payload)
```